mmv1/templates/terraform/constants/network_services_gateway.go.tmpl

```diff
@@ -10,6 +10,15 @@
 	See the License for the specific language governing permissions and
 	limitations under the License.
 */ -}}
+
+import (
```

Comment:
Do you need to conditionally use different packages here? For example, project just uses the ga wait function: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/resourcemanager/resource_google_project.go#L18

Reply:
If I use only the GA, when running on Beta it will show this error:
```
cannot use config (variable of type *"github.com/hashicorp/terraform-provider-google-beta/google-beta/transport".Config) as *"github.com/hashicorp/terraform-provider-google/google/transport".Config value in argument to tpgcompute.ComputeOperationWaitTime
```